src/messages.ts

```diff
@@ -52,5 +52,10 @@ export const messages = {
         failure: 'Unable to generate data to release, please check the widget template config or template data',
         invalidName: 'Please provide a valid widget-template',
     },
+    widgetDelete: {
```

Comment:
Could we rename this to `deleteWidgetTemplate`?

Reply:
Sure thing.